slack_notifier.go

```diff
@@ -22,6 +22,7 @@ type Payload struct {
 	IconEmoji   string       `json:"icon_emoji"`
 	Username    string       `json:"username"`
 	Channel     string       `json:"channel"`
+	Thread      string       `json:"thread_ts,omitempty"`
```

Comment:
@kseniyashaydurova  thanks for the PR.
Why `thread_ts` here and not just `thread` ?